BaristaSwift/Common/Common/Base/Store.swift

```diff
@@ -30,8 +30,7 @@ import SalesforceSDKCore
 import SalesforceSwiftSDK
 import SmartStore
 import SmartSync
-
-public typealias SyncCompletion = ((SFSyncState?) -> Void)?
```

Comment:
Every function that had that completion block now returns a Promise<Void> instead.